packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -34,6 +34,223 @@ export interface HttpRouteMatch {
    * and you want the route to match requests to my-service.local/metrics, your prefix should be /metrics.
    */
   readonly prefixPath: string;
+
+  /**
+   * Specifies the client request headers to match on. All specified headers
+   * must match for the route to match.
+   *
+   * @default - do not match on headers
+   */
+  readonly headers?: HeaderMatch[];
+
+  /**
+   * The client request method to match on. Specify only one.
+   *
+   * @default - do not match on request method
+   */
+  readonly method?: HttpRouteMatchMethod;
+
+  /**
+   * The client request scheme to match on. Specify only one. Applicable only for HTTP2 routes.
+   *
+   * @default - do not match on HTTP2 scheme
+   */
+  readonly scheme?: HttpRouteMatchScheme;
+}
+
+/**
+ * Supported request matching methods.
```

Comment:
This reads a little awkward. How about: "Supported values for matching routes based on the HTTP request method"?